src/adapters/supabase/helpers/client.ts

```diff
@@ -2,6 +2,7 @@ import { createClient, SupabaseClient } from "@supabase/supabase-js";
 import { getAdapters, getLogger } from "../../../bindings";
 import { Issue, UserProfile } from "../../../types";
 import { Database } from "../types";
+import { BigNumber, BigNumberish } from "ethers";
```

Comment:
You can represent BigInts natively in TypeScript by appending `n` at the end of the number. Is importing this library necessary nowadays?

@rndquu @molecula451 rfc

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt

Reply:
To eliminate this issues we should consider layout a contributions template of code conventions, to what use and when and what not(look for google C++ code convention as an e.g, also VirtualBox code convention, they got inspired by the google one, even Mozilla) this would help ups keep the project with good code consistency.

We should use BigInt when handling normal BigInts operations
We should use BigNumber from ethers packages when handling EVM operations.

looks like we're handling a lot of EVM stuff including `accounts[0]` deduction from `BigNumber`, etc so it looks good

Reply:
`BigInt` is for integer values while `BigNumber` is for arbitrary-precision floating point values so `BigNumber` is the correct choice